### PR TITLE
Adds note about DTLS over ICE and "5-tuples".

### PR DESCRIPTION
**Pull request by alvestrand**

### CODE REVIEW

File: rtcweb-transports.xml [hunk 1]
```diff
@@ -214,6 +214,13 @@
         <t>The setup protocol for WebRTC data channels is described in <xref
         target="I-D.ietf-rtcweb-data-protocol"/>.</t>
 
+        <t>Note: DTLS-SRTP as defined in <xref target="RFC5764"/> defines
+        protection of data carried over a single UDP source and destination
+        port pair (RFC 5764 section 3). In the context of using ICE, the term
+        "single UDP source and destination port pair" needs to be understood
+        as "a single ICE component", as defined in <xref target="RFC5245"/>.
+        This note may be included in a future version of DTLS-SRTP or ICE.</t>
```
Comment (ekr) [inline rtcweb-transports.xml]: I would strike this last sentence.

**Comment by ekr** [comment]
LGTM with comment above.

**Comment by juberti** [comment]
I think we should just refer people to 5763, S 6.7.1:

   Interactive Connectivity Establishment (ICE), as specified in
   [RFC5245], provides a methodology of allowing participants in
   multimedia sessions to verify mutual connectivity.  When ICE is being
   used, the ICE connectivity checks are performed before the DTLS
   handshake begins.  Note that if aggressive nomination mode is used,
   multiple candidate pairs may be marked valid before ICE finally
   converges on a single candidate pair.  Implementations MUST treat all
   ICE candidate pairs associated with a single component as part of the
   same DTLS association.  Thus, there will be only one DTLS handshake
   even if there are multiple valid candidate pairs.  Note that this may
   mean adjusting the endpoint IP addresses if the selected candidate
   pair shifts, just as if the DTLS packets were an ordinary media
   stream.

Specifically, I think we should say:

Note: DTLS-SRTP as defined in <xref target="RFC5763"/> defines how DTLS interacts with ICE in Section 6.7.1. As indicated therein, implementations MUST treat all ICE candidate pairs associated with a single component as part of the same DTLS association.  Thus, there will be only one DTLS handshake even if there are multiple valid candidate pairs.

**Comment by alvestrand** [comment]
Rephrased according to Justin's suggestion.

**Pull request merged**